### ISSUE TITLE
Names shouldn't start with /

### ISSUE THREAD

**Issue by Shereef** [DDRace/teeworlds#187]
In Teeworlds / means nothing but in DDRace it does, so for people who have their names starting with / the server should replace it with _ or just shifts it 1 character and add a space in the beginning, just so that tab completion works in the beginning of a sentence

**Comment by heinrich5991** [comment]
not sure if that wuold be a good idea, could cause some conflicts.

if someone has a name like '/mephisto\', '_mephisto\' would look ugly. If someone uses the full name length and everything gets shifted (not even sure whether it would work, maybe the client strips starting whitespace), the name would be shortened.

**Comment by Shereef** [comment]
I think the rename with a warning chat message is the way to go, or just
remove all forward slashes from start of name and warn

sent from my Galaxy Nexus (Android doesn't support HTML email formatting)
On Jul 24, 2012 12:27 PM, "heinrich5991" <
reply@reply.github.com>
wrote:

**Comment by Tsin** [comment]
btw maybe new "punishment" ^^

id 3 = nameless tee

"rename 3(id) "i love cows"

and a message 

player "nameless tee" was renamed to "i love cows"

this would be better because it makes a good laugh of a bad player whole server can laugh and giving bans/mutes etc dont rly work in this game :P

yeah maybe i should start a new issue but rly dont know if you will approve it and spamming this git with issues would be a little bit annoying

**Comment by heinrich5991** [comment]
I completely disagree with this feature, it will definitely be abused by kiddy admins.

**Comment by Tsin** [comment]
first of all why do you care about kiddy admins 

lets face it normal people play on "trusted" servers like procodex ddracemax or that russian server "konat4"

other servers are all with iddrace or xllddrace aka for nooby kiddy admins so who cares about them

**Comment by Shereef** [comment]
Tsin, we don't want to give the admin too much control.
It's either a server feature that is unmodifiable or it's not there at all, because when an admin abuses his authority DDRace name gets run down to the ground, not him.

**Comment by Tsin** [comment]
so lets see a kid with nickname 

Penis or Fuck You or Your Mother Sucks Dicks whatever plays 

now i can mute him (oh wait he wont care)
i will give him ban 
everyone on server will act like "why did you give him ban"

or rename his nickname to "I love cows"

**Comment by heinrich5991** [comment]
just ban him - simple stuff

**Comment by Learath2** [comment]
Muted players shouldnt be able to change names in my opinion..

**Comment by Tsin** [comment]
agree with one exception players muted by admin shouldnt be able to change names

but dont touch the players muted by server

**Comment by eeeee** [comment]
If it was up to me, i'd allow only latin letters and numbers in nicknames, coz its annoying when some kid has only UTF shit in nickname and you cant even refer to him in chat.

**Comment by ravomavain** [comment]
Why not simply refuse the connection or immediately kick (depending on when the nick is available to the server) player whose name start with / with a kick message like "Nickname shouldn't start with /" ?
So it avoid nickname conflicts (which could also be handled by adding (1) at the end like we do for nameless tee) and player won't ask questions like "why my nickname have been changed".
